wallets/react-wallet-v2/src/data/RelayerRegions.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Types
+ */
+
+type RelayerType = {
+    value: string ;
+    label: string;
+};
+
+/**
+ * Relayer Regions
+ */
+export const REGIONALIZED_RELAYER_ENDPOINTS: RelayerType[] = [
```

Comment:
we also have staging (`staging.relay.walletconnect.com` and `https://staging.us-east-1.relay.walletconnect.com/hello` etc). But that can be in done in a dedicated PR

Reply:
Will add to `dev` in another PR.